lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -438,11 +437,7 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            if host[:type] == 'aio'
```

Comment:
@melissa since we're no longer supporting Puppet 3.x, it seems like we should always be setting `host[:type]` to `aio`. Unfortunately, Beaker defaults to `pe` as its type, so there's definitely a chance that this won't be the case as people slip through the cracks. Our testing should help give us some confidence in this change 😺